packages/lit-dev-content/src/components/litdev-tutorial-card.ts

```diff
@@ -0,0 +1,280 @@
+      /* Break out of root padding */
+      margin-inline: calc(var(--_unit) * -0.5);
+    }
+
+    img {
+      width: 100%;
+    }
+
+    #header {
+      font-family: Manrope;
+    }
+
+    #header ::slotted(*) {
+      font-weight: normal;
+      margin: 0;
+      font-size: calc(var(--_unit));
+    }
+
+    #metadata-container {
+      /* TODO(e111077): change to theme custom prop */
+      color: #6f6f6f;
+      display: flex;
+      flex-grow: 1;
+      flex-direction: column;
+      justify-content: space-between;
+      flex-direction: column-reverse;
+    }
+
+    #metadata {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    #description,
+    #metadata {
+      font-family: 'Open Sans', sans-serif;
+      font-size: calc(var(--_unit) * 0.5);
+    }
+
+    #description {
+      overflow-y: hidden;
+    }
+
+    /* Markdown renderer will wrap content in <p> tags. Remove margins on the
+    first and last */
+    #description ::slotted(:first-of-type) {
+      margin-block-start: 0;
+    }
+
+    #description ::slotted(:last-of-type) {
+      margin-block-end: 0;
+    }
+
+    :host::before {
+      content: '';
+      position: absolute;
+      /* Below the content */
+      z-index: -1;
+      inset: 0;
+      transform: translate(0, 0);
+      background-color: var(--color-blue);
+    }
+
+    #root,
+    :host::before {
+      transition: transform 100ms ease-out;
+    }
+
+    :host(:hover) #root,
+    :host(:focus-within) #root {
+      /* traslate the root / content to up right */
+      transform: translate(
+        calc(var(--_unit) * 0.125),
+        calc(var(--_unit) * -0.125)
+      );
+    }
+
+    :host(:hover)::before,
+    :host(:focus-within)::before {
+      /* translate the background down left */
+      transform: translate(
+        calc(var(--_unit) * -0.125),
+        calc(var(--_unit) * 0.125)
+      );
+    }
+
+    @media (prefers-reduced-motion) {
+      :host::before,
+      #root {
+        transition: none;
+      }
+    }
+  `;
+
+  /**
+   * Which card size variant to use:
+   * tiny = 1x0.5
+   * small = 1x1
+   * medium = 1x1.5
+   * large = 1x2
+   */
+  @property({type: String, reflect: true}) size: CardSize = 'small';
+
+  /**
+   * Optional src for the image
+   */
+  @property({attribute: 'img-src'}) imgSrc = '';
+
+  /**
+   * Alt text for the image;
+   */
+  @property({attribute: 'img-alt'}) imgAlt = '';
+
+  /**
+   * Difficulty of the tutorial
+   */
+  @property() difficulty = 'Beginner';
+
+  /**
+   * Duration of the tutorial in minutes
+   */
+  @property({type: Number}) duration = 0;
+
+  /**
+   * Whether or not to hide the description. Used when there is no description
+   * and we don't what the whitespace associated with the node.
+   */
+  @property({type: Boolean, attribute: 'hide-description'}) hideDescription =
+    false;
+
+  /**
+   * Link to tutorial
+   */
+  @property() href = '';
+
+  render() {
+    return html` <a id="root" href=${this.href}>
```

Comment:
Does automatic formatting add these single spaces between the opening backtick and the <a? No action required.

Reply:
ugh yes it's so gross